cmake/MarlinMacros.cmake

```diff
@@ -246,7 +247,13 @@ function( MARLIN_ADD_PROCESSOR_TEST test_name )
   # configure cmake test file.
   # Read the file content and use file( GENERATE )
   # because the MARLIN_DLL contains generator expressions
-  file( READ ${PROJECT_SOURCE_DIR}/cmake/runmarlin.cmake.in TEST_FILE_CONTENT )
+	set (RUN_INPUT_FILE "${PROJECT_SOURCE_DIR}/cmake")
+	if (ARG_PARALLEL)
```

Comment:
It would be better to pass the cmake target in script configuration instead of duplicating a cmake script for a change of a few characters.

Reply:
sure, introduce PROCESSOR_TEST_TARGET